src/app.ts

```diff
@@ -10,6 +10,7 @@ import {
   updateTokenSecret,
```

Comment:
Please drop changes to this file from this PR. You're doing the same changes already in #9, however the changeset already diverged. This PR will have conflict on this file if #9 gets merged.

Reply:
Should be dropped now